io.elementary.Sdk.json

```
@@ -183,6 +209,28 @@
                     "tag": "1.2.0"
                 }
             ]
+        },
+        {
+            "name" : "platform-settings",
+            "buildsystem" : "simple",
+            "modules": [
+                {
+                    "name" : "default-gtk-settings",
+                    "buildsystem" : "simple",
+                    "sources" : [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/elementary/default-settings.git"
+                        }
+                    ],
+                    "build-commands": [
+                        "install -Dm644 gtk/settings.ini /etc/gtk-3.0/settings.ini"
```

I think it would be better if it was installed `/usr/etc` instead. I'd be surprised if `/etc` made it to the compose.

```suggestion
                        "install -Dm644 gtk/settings.ini /usr/etc/gtk-3.0/settings.ini"
```